tlspuffin/src/protocol.rs

```diff
@@ -339,7 +341,8 @@ impl Default for TLSDescriptorConfig {
             server_authentication: true,
             try_reuse: false,
             typ: AgentType::Server,
-            cipher_string: String::from("ALL:!EXPORT:!LOW:!aNULL:!eNULL:!SSLv2"),
+            cipher_string_tls13: String::from("ALL:!EXPORT:!LOW:!aNULL:!eNULL:!SSLv2"),
```

Comment:
Suggestion: use a global const string instead of inlining here

Reply:
Done